src/common/components/repository-row/dropdowns/register-name-dropdown.js

```diff
@@ -47,6 +54,60 @@ const Caption = (props) => {
   );
 
   let caption;
+  let changeForm;
+  let message;
+
+  const changeRegisteredName = !!registeredName;
+
+  // only show "change name" info and form if there is a name registered already,
+  // user is authenticated and we are not showing register success message
+  const showChangeForm = (
+    changeRegisteredName &&
+    authStore.authenticated &&
+    !registerNameStatus.success
+  );
+
+  if (!changeRegisteredName) {
```

Comment:
I usually find this kind of conditional easier to read when the positive case comes first.